src/Simplex/Messaging/Agent.hs

```diff
@@ -249,6 +248,12 @@ processSMPTransmission c@AgentClient {sndQ} (srv, rId, cmd) = do
             A_MSG body -> do
               logServer "<--" c srv rId "MSG <MSG>"
               -- TODO check message status
+              withStore $ \st -> createRcvMsg st connAlias body senderMsgId senderTimestamp srvMsgId srvTs
+              -- TODO either:
+              -- TODO   - pass MSG{.., msg = generatedRcvMsg} to notify,
+              -- TODO   - retrieve internalTs from generatedRcvMsg,
+              -- TODO   - pass recipientTs to createRcvMsg (second cleanest),
+              -- TODO   - remove m_recipient from MSG (might be the cleanest?).
```

Comment:
Internal ID is needed to refer to the messages, but we can change the syntax later

Reply:
let's discuss which approach to take here - will change in separate PR